tests/foreman/cli/test_repository.py

```diff
@@ -304,6 +352,37 @@ def test_negative_create_with_name(self):
                 with self.assertRaises(CLIFactoryError):
                     self._make_repository({u'name': name})
 
+    @tier1
+    def test_negative_create_with_auth_url_forbidden_characters(self):
+        """Verify that repository URL cannot contain forbidden characters
+
+        @Feature: Repository
+
+        @Assert: Repository cannot be created
+        """
+
+        for cred in [creds for creds in valid_http_credentials()
```

Comment:
Is it right to use `valid_http_credentials()` method for negative scenario which has phrase `cannot contain forbidden characters` in the docstring?